back/src/Model/Group.ts

```diff
@@ -106,6 +106,16 @@ export class Group implements Movable {
         return this.users.size <= 1;
     }
 
+    includes(user: User): boolean {
+        let found = false;
+        this.users.forEach((currentUser: User) => {
+            if (currentUser.name === user.name) {
+                found = true;
+            }
+        });
+        return found;
+    }
```

Comment:
This `includes` method actually looks unused. Maybe a forgot to remove it while refactoring?
I think you can safely remove it.

Reply:
Indeed. Will do.